apps/frontend/src/libs/enums/notification-message.enum.ts

```diff
@@ -1,4 +1,6 @@
 const NotificationMessage = {
+	ACCESS_DENIED:
+		"You don't have access to any pages for now. Please contact an admin to enable access for you.",
```

Comment:
```suggestion
		"You do not have permission to access this page. Please contact an administrator to request access.",
```

Reply:
We need to coordinate this with QA since Veronika wrote this text for the notification.

Reply:
Oh this is totally fine, I have written the basic logic. You can take the one that Joel have suggested